src/olympia/lib/settings_base.py

```diff
@@ -1121,9 +1144,6 @@ def JINJA_CONFIG():
         'queue': 'addons'},
     'olympia.addons.tasks.version_changed': {'queue': 'addons'},
 
-    # Files (goes to devhub queue).
-    'olympia.files.tasks.update_webext_descriptions_all': {'queue': 'devhub'},
```

Comment:
rebased into two commits; the latter that only removes a queue from `CELERY_ROUTES` to test